controllers/pipelines/suite_decoupled_test.go

```diff
@@ -89,7 +90,7 @@ var _ = BeforeSuite(func() {
 	testConfig = config.Configuration{
 		DefaultExperiment: "Default",
 		WorkflowNamespace: "default",
-		DefaultProvider:   apis.RandomString(),
+		DefaultProvider:   strings.ToLower(apis.RandomString()),
```

Comment:
This might become quite common? What do you think about actually providing the function RandomLowercaseString or something in the test helper?

Reply:
already existing just hadn't seen it, changed in [f14cca8](https://github.com/sky-uk/kfp-operator/pull/354/commits/f14cca83d6fe1a7cb015482b23effbdc540042ee)